ci/tasks/teardown.sh

```
@@ -0,0 +1,23 @@
+REPO_PARENT="$( cd "${REPO_ROOT}/.." && pwd )"
+
+if [[ -n "${DEBUG:-}" ]]; then
+  set -x
+  export BOSH_LOG_LEVEL=debug
+  export BOSH_LOG_PATH="${BOSH_LOG_PATH:-${REPO_PARENT}/bosh-debug.log}"
+fi
+
+mv "${REPO_PARENT}"/director-state/* "${REPO_PARENT}/"
+mv "${REPO_PARENT}/director-state/.bosh" "${HOME}/"
+
+export BOSH_ENVIRONMENT=`bosh int "${REPO_PARENT}/director-creds.yml" --path /internal_ip`
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)